experimental/beacon_sim/belief_road_map_planner.cc

```diff
@@ -185,6 +185,28 @@ double uncertainty_size(const RobotBelief &belief) {
     return belief.cov_in_robot.determinant();
 }
 
+double uncertainty_size(const LandmarkRobotBelief &belief) {
+    std::vector<LandmarkRobotBelief::LandmarkConditionedRobotBelief> elements;
+    elements.reserve(belief.belief_from_config.size());
+    for (const auto &[_, value] : belief.belief_from_config) {
+        elements.push_back(value);
+    }
+    std::sort(elements.begin(), elements.end(), [](const auto &a, const auto &b) {
+        return a.cov_in_robot.determinant() < b.cov_in_robot.determinant();
+    });
+
+    constexpr double EVALUATION_THRESHOLD = 0.95;
+
+    double accumulated_prob = 0.0;
+    for (const auto &elem : elements) {
+        accumulated_prob += std::exp(elem.log_config_prob);
+        if (accumulated_prob > EVALUATION_THRESHOLD) {
+            return elem.cov_in_robot.determinant();
+        }
+    }
+    return elements.back().cov_in_robot.determinant();
```

Comment:
Is returning the last element in the cases where 95% probability isn't reached the right behavior here? If the elements of a robot belief don't cover all configurations it might be hard to debug why the sizes are smaller than expected.

Reply:
added a check